### PR TITLE
#168361702 User can sign up 

### PR DESCRIPTION
**Pull request by sharonandisi**
#### What does this PR do?
This creates the user can sign up endpoint
#### Description of Task to be completed?
A user should be able to sign up
#### How should this be manually tested?
By cloning this [repo](https://github.com/sharonandisi/FreeMentorsv2) and following the instructions in the readme to set up.
#### What are the relevant pivotal tracker stories?
[#168361702](https://www.pivotaltracker.com/n/projects/2379605)
#### Screenshots
Screenshot from 2019-09-10 05-25-02

### CODE REVIEW

**Comment by ekumamait** [comment]
The response should be `"data": { "token": "4538ebdqei92e024"}` only and nothing else and why is the build failing?

File: server/tests/usertest.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+chai.should();
+chai.use(chaiHttp);
+
+describe('/Auth', () => {
+    describe('/POST signup', () => {
+        it('should successfully sign up a user', (done) => {
+            chai.request(app)
+                .post('/api/v1/auth/signup')
+                .send({
+                    firstname: 'sharon',
```
Comment (ekumamait) [inline server/tests/usertest.js]: @sharonandisi please create a separate folder in your test folder and name it mockData inside it create a file for example called i.e userMocks.js. inside it, you will create instances of data to be used in your test file. It is not ethical to write the data in the test file. rather import the data from another file. please use this in all scenarios including for tokens.

**Review by ekumamait — COMMENTED** [review]
Take a look at this as well

File: server/tests/usertest.js (hunk 1 above)
Comment (sharonandisi) [reply server/tests/usertest.js]: Alright. Noted. Am on it

Comment (sharonandisi) [reply server/tests/usertest.js]: I don't know why one test is failing on travis while they all pass locally . Let me look into it as I make the changes you have sited

**Pull request merged**